src/components/checkbox/checkbox.test.tsx

```diff
@@ -114,6 +114,35 @@ test("should render tooltip with provided labelHelp and helpAriaLabel", async ()
   expect(helpTooltip).toHaveAccessibleDescription("labelHelp");
 });
 
+test("should render input with validation tooltip as its accessible description when the input is focused", async () => {
```

Comment:
**praise**: good idea to include this so the intended outcome of handling the element ids is covered 👍🏼